docs/src/contribute/code.md

```diff
@@ -10,7 +10,7 @@ If you plan to make code changes to this repository, you will need to install th
 
 Once done, you should clone this repository:
 
-```bash
+```bash linenums="0"
```

Comment:
Is there any way to create a macro or configure this to be the default. I feel like enforcing this as a consistent style will be tedious.

More of an observation than a requirement for merging this.

Reply:
That ironically is the default behavior. But at some point I realized we had way more code examples than bash examples, so I changed the default to enable line numbers to show.

I can change the default back, but then I'd need to manually write `linenums=1` on way more LOCs.